charts/prometheus-node-exporter/README.md

```diff
@@ -41,6 +41,10 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 3.0.0
+
+￼Due to a change in DaemonSet labels, the upgrade requires `helm upgrade --force` in order to re-create the daemonset.
```

Comment:
Thats not completlay true.
You also can do it like this: https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/README.md#from-chart-versions--500